app/views/submissions/_submissions_table.js.jsx.erb

```diff
@@ -188,9 +195,9 @@
         var s = {};
         s['id'] = submission.id;
         if (submission.name_url === '') {
-          s['group_name'] = <span>{submission.name}</span>;
+          s['group_name'] = <span dangerouslySetInnerHTML={{__html: submission.name}} />;
```

Comment:
That's the attribute it was looking for in javascripts/ReactComponents/table.js. I guess I should be changing what it's looking for there as well?